README.md

```diff
@@ -100,6 +100,70 @@ docker run -d --restart on-failure --name iotex \
 
 6. Make sure TCP ports 4689, 8080 (also 14014 if used) are open on your firewall and load balancer (if any).
 
+## <a name="mainnet_native"/>Join MainNet Beta on Linux
+
+1. Set the environment with the following commands:
```

Comment:
to avoid copies, let's do this 
-- steps 1,2,3,4 are the same as 
https://github.com/iotexproject/iotex-bootstrap/blob/master/README.md#mainnet
-- step 5 - "Build server binary"...
-- step 6 - "Run the following command to start a node"...
-- step 7 - "Make sure TCP ports 4689"...